### ISSUE TITLE
Get ordered data

### ISSUE THREAD

**Issue by fcofabricio** [handsontable/handsontable#989]
How I get the ordered data after sorting operation?

**Comment by MikeMajara** [comment]
This still does not work. The workaround exposed by @psmolenski does not guarantee you get the data as displayed. Is there another workaround?

**Comment by AMBudnik** [comment]
Hi @MikeMajara Does this solution meet your requirements http://jsfiddle.net/5q8ketrj/?

**Comment by MikeMajara** [comment]
Nope... The problem for me happens to be after using `hot.getPlugin('columnSorting').sort(0, 'asc');`. I apply this sorting each time a cell is edited in order to keep the rows sorted. After, I want to apply a different order algorithm for which I need the data sorted, but it seems to be unsorted internally although it is shown otherwise on the table.

The workaround I found (although it might not be suitable for all) is sorting the source data and rendering. It is not Ideal because I have to keep elsewhere the original order. But it seems to do the job. 

In fact, hot.getData() with or without parameters actually does return data ordered as shown once the plugin sorting is removed.

TL;DR;

columnSorting plugin is messing up with the order related to what is visible on the table. Use `Array.prototype.sort()` as specified [here](https://docs.handsontable.com/3.0.0/demo-sorting.html).
